internal/cli/server.go

```diff
@@ -31,6 +32,14 @@ func init() {
 	flags.Int("port", 0, "(PORT) The port to listen on. Leave empty to auto-assign a random port")
 	viper.BindPFlag("port", flags.Lookup("port"))
 	viper.BindEnv("port", "PORT")
+
+	flags.String("tls-issuer", deployments.SelfSignedIssuer, "(TLS_ISSUER) The cluster issuer to use for workload certificates")
+	viper.BindPFlag("tls-issuer", flags.Lookup("tls-issuer"))
+	viper.BindEnv("tls-issuer", "TLS_ISSUER")
+
+	flags.Bool("use-internal-registry-node-port", true, "(USE_INTERNAL_REGISTRY_NODE_PORT) Use the internal registry via a node port")
```

Comment:
There is no point int setting this to true if `enable-internal-registry-node-port` was set to `false` at installation time right?
It seems to me, the only reason we need this flag is because we have no place to store the value of the `enable-internal-registry-node-port` flag that was used. Is there a way to detect what value was used? For example, check the registry service to see what type it is?

Reply:
How does the user define this for the server? Epinio sets up the server command automatically inside the server deployment, the user doesn't have a way to provide flags. The install command is the one that creates the server command and that one has access to the `--enable-internal-registry-node-port` flag. Maybe we should use that to define the `--use-internal-registry-node-port`?

Reply:
Yes, that was the plan. Let me check if I forgot to do this...

Reply:
Ah here it is, some awful indirection, but all variable names match the pattern (snake case, kebap case, ...)

https://github.com/epinio/epinio/pull/590/files#diff-892d335f47fd05c8157a81b96c028b3d6c1a88492ca015135d24fe86a1ede046R308-R309

So the client puts this as an env var into the epinio deployment yaml via our ##substitution##. (I feel we should invest in real templates soon;)

I thought 'use' was more approriate for the server flag, but maybe it should be the same name as the client flag 'enable' ?

Reply:
maybe `use` should be used in all cases? I mean, they use it in a different way but they both use it somehow. Otoh, I'm still thinking we could get rid of the server flag but inspecting the registry service. It's only used in one place (the staging api) in order to decide what the registry URL should look like.